src/main/java/gui/misc/TransitionHelper.java

```diff
@@ -25,20 +27,36 @@
 
     @Getter
     private Node node;
+    @Getter
+    private ArrayList<EventHandler> addedHandlers;
```

Comment:
`private List<EventHandler> addedHandlers;`
and
`this.addedHandlers = new ArrayList<EventHandler>();`
So fixed.